cluster/vmpreset-windows.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows2012r2-preset
+  selector:
+    matchLabels:
+      flavor: windows2012r2
+spec:
+  terminationGracePeriodSeconds: 0
+  domain:
+    cpu:
+      cores: 2
+    features:
+      acpi: {}
+      apic: {}
+      hyperv:
+        relaxed: {}
+        vapic: {}
+        spinlocks:
+          spinlocks: 8191
+    clock:
+      utc: {}
+      timer:
+        hpet:
+          present: false
+        pit:
+          tickPolicy: delay
+        rtc:
+          tickPolicy: catchup
+        hyperv: {}
+    firmware:
+      uuid: 5d307ca9-b3ef-428c-8861-06e72d69f223
+    resources:
```

Comment:
can you please move the resources up close to the cpu part - then it's easier to read.